packages/scripts/src/monitor-oa/MonitorLogger.ts

```diff
@@ -0,0 +1,96 @@
+import { createEtherscanLinkMarkdown, createFormatFunction } from "@uma/common";
```

Comment:
I like this pattern! pull out the "formatting" work from the logger into this separate file then use it within separate "Monitor" modules that check for different things (Dispute, Assertion, Settlement).